debian/control

```diff
@@ -653,12 +661,13 @@ Description: RocksDB storage engine for MariaDB
 
 Package: mariadb-plugin-columnstore
 Architecture: amd64 i386
-Depends: mariadb-server-10.5 (= ${binary:Version}),
-         net-tools,
-         binutils,
-         libsnappy1 | libsnappy1v5,
+Depends: binutils,
+         expect,
          libjemalloc1 | libjemalloc2,
-         python | python2 | python3,
+         libsnappy1 | libsnappy1v5,
+         mariadb-server-10.5 (= ${binary:Version}),
+         net-tools,
+         python,
```

Comment:
After a lot of thinking we decide this will be python3 since the next release not python2.

Reply:
This references automatically the latest Python version. You cannot use Python 2 on new software anymore, it has been removed from Ubuntu and Debian.

Related, please update all scripts to have the header `#!/usr/bin/python3`, don't use deprecated env thing.